versel.json

```diff
@@ -0,0 +1,29 @@
+# versel.json
```

Comment:
Should the file name be `versel.json` or `vercel.json`? Also technically a comment in JSON does not work.